content/applications/inventory_and_mrp/inventory/management/delivery/dropshipping.rst

```diff
@@ -1,109 +1,80 @@
-the *Route* field appear on the sale order lines to specify you send a
-product via drop-shipping.
+Click on the :guilabel:`Purchase` tab and specify a vendor and the price that they sell the product
+for. Multiple vendors can be added, but the vendor at the top of the list will be the one
+automatically selected for purchase orders.
 
-.. image:: dropshipping/dropshipping_02.png
-    :align: center
+.. image:: dropshipping/product-vendor-config.png
+   :align: center
+   :alt: The product form with a vendor specified.
 
-Now, in the *Sales* app, go to :menuselection:`Products --> Products`. Select the
-product you would like to drop-ship and add a vendor pricelist which
-contains the right supplier, via the purchase tab.
+Finally, select the :guilabel:`Inventory` tab and enable the :guilabel:`Dropship` checkbox in the
+:guilabel:`Routes` section.
 
-.. image:: dropshipping/dropshipping_03.png
-    :align: center
+.. image:: dropshipping/enable-dropship-route.png
+   :align: center
+   :alt: Enable the Dropship option in the product inventory tab.
 
-Send Products from the Suppliers Directly to the Customers
-==========================================================
+Fulfill orders using dropshipping
+=================================
 
-Create a *Sales Order* and add the product on which you just set the
-vendor. Add the *Route* field thanks to the widget on the right of the
-sale order line. Now, you are able to specify that your products’ route
-is *Drop-Shipping*.
+When a customer completes an online purchase for a dropshipped product, a sales order will be
+automatically created with an associated request for quotation (:abbr:`RFQ (Request for Quotation)`)
+for the supplier. Sales orders can be viewed in the :guilabel:`Sales` app by selecting
+:menuselection:`Orders --> Orders`. Click the :guilabel:`Purchase` smart button at the top right of
+a sales order to view the associated :abbr:`RFQ (Request for Quotation)`.
 
-.. image:: dropshipping/dropshipping_04.png
-    :align: center
+.. image:: dropshipping/dropship-sales-order.png
+   :align: center
+   :alt: A dropship sales order with the Purchase smart button in the top right corner.
 
-.. image:: dropshipping/dropshipping_05.png
-    :align: center
+.. tip::
+   Dropship sales orders can also be created manually through the :guilabel:`Sales` app by selecting
+   :menuselection:`Orders --> Orders --> Create` and adding a product configured for dropshipping.
 
-Once the sale order is confirmed, Odoo automatically generates a
-*Request for Quotation* for the supplier who will proceed to the
-drop-shipping. You can find it in the *Purchase* app, with the sale
-order as *Source Document*.
+Once the :abbr:`RFQ (Request for Quotation)` is confirmed, it becomes a purchase order and a
+dropship receipt is created and linked to it. The receipt can be viewed by clicking the
+:guilabel:`Receipt` smart button at the top right of the purchase order.
```

Comment:
Slightly more navigational context...
You'd be surprised how often smart buttons get missed 😅 
```suggestion
:guilabel:`Receipt` smart button in the top right corner of the purchase order form.
```